element-hq/hydrogen-web
# Fixed autocomplete bug by enclosing div within a form tag #380

> Description

For issue #380, I have inserted a `<form>` that is wrapped around the `<div>` as it is needed for the autocomplete as given at [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) so the `utilitiesRow` variable now returns this `<form>`.
I have assigned a name attribute of `utilities-form` to the `<form>`.
> Before:

![image](https://user-images.githubusercontent.com/64756497/145767786-99f66339-1d68-486c-9ecc-189a988d32d7.png)

> After:

![image](https://user-images.githubusercontent.com/64756497/145767603-61924a49-7cde-4e44-8b15-9cec20bff13e.png)

> Testing Change

1. Save a login credential on the website on google chrome
2. Open settings to check whether the username is autofilled on click in room filter search and password for the security field.


Resolves #380 

Signed-off-by: Aaryan Gautam <aaryanyo123@gmail.com>
